reinforcement_learning/common/sagemaker_rl/ray_launcher.py

```diff
@@ -97,7 +98,15 @@ def customize_experiment_config(self, config):
         # TODO: move this to before customer-specified so they can override
         hyperparams_dict["rl.training.local_dir"] = INTERMEDIATE_DIR
         hyperparams_dict["rl.training.checkpoint_at_end"] = True
-        hyperparams_dict["rl.training.checkpoint_freq"] = 10
+
+        try:
+            customized_checkpoint_frequency = config['training']['checkpoint_freq']
```

Comment:
You can use `customized_checkpoint_frequency = config['training'].get('checkpoint_freq', 10)` to set default value for `checkpoint_freq `. In that way I guess you can bypass the `try except`

Reply:
Good point.